### DISCUSSION TITLE
Where does the css actually get injected?

### DISCUSSION THREAD

**Discussion by alexanderhorner** [chakra-ui/panda#1162] [Q&A]
When I inspect my elements and look where the css properties come from, it just points me to the global.is file where I defined my layers. 

Where does the generated css actually go? I'm not importing anything, so what's going on?

**Comment by astahmer** [comment]
you are probably using the postcss plugin, which handles that part
otherwise, if you're using the CLI only, then you can import a CSS file generated after each `panda` or `panda cssgen` command run by yourself

**Reply by alexanderhorner** [reply]
> you are probably using the postcss plugin, which handles that part
> 
> otherwise, if you're using the CLI only, then you can import a CSS file generated after each `panda` or `panda cssgen` command run by yourself

Yes I'm using the post css plug-in, but the generated css classes have to be in some file that's imported or inline somewhere right?

**Reply by astahmer** [reply]
I detailed that [here](https://github.com/chakra-ui/panda/discussions/1034#discussioncomment-6538208)
